alembic_migration/versions/24f8da659c78_add_ratings_to_outings.py

```diff
@@ -0,0 +1,360 @@
+                              'P4+', name='equipment_rating',
+                              schema='guidebook'),
+                      nullable=True),
+            schema='guidebook')
+
+        op.add_column(
+            table,
+            sa.Column('ice_rating',
+                      sa.Enum(
+                          '1', '2', '3', '3+', '4', '4+', '5', '5+', '6', '6+',
+                          '7', '7+',
+                          name='ice_rating',
+                          schema='guidebook'),
+                      nullable=True),
+            schema='guidebook')
+
+        op.add_column(
+            table,
+            sa.Column('rock_free_rating',
+                      sa.Enum(
+                          '2', '3a', '3b', '3c', '4a', '4b', '4c', '5a', '5a+',
+                          '5b', '5b+', '5c', '5c+', '6a', '6a+', '6b', '6b+',
+                          '6c', '6c+', '7a', '7a+', '7b', '7b+', '7c', '7c+',
+                          '8a', '8a+', '8b', '8b+', '8c', '8c+', '9a', '9a+',
+                          '9b', '9b+', '9c', '9c+',
+                          name='climbing_rating', schema='guidebook'),
+                      nullable=True),
+            schema='guidebook')
+
+        op.add_column(
+            table,
+            sa.Column('via_ferrata_rating',
+                      sa.Enum('K1', 'K2', 'K3', 'K4', 'K5', 'K6',
+                              name='via_ferrata_rating', schema='guidebook'),
+                      nullable=True),
+            schema='guidebook')
+
+        op.add_column(
+            table,
+            sa.Column('hiking_rating',
+                      sa.Enum('T1', 'T2', 'T3', 'T4', 'T5',
+                              name='hiking_rating', schema='guidebook'),
+                      nullable=True),
+            schema='guidebook')
+
+        op.add_column(
+            table,
+            sa.Column('snowshoe_rating',
+                      sa.Enum('R1', 'R2', 'R3', 'R4', 'R5',
+                              name='snowshoe_rating', schema='guidebook'),
+                      nullable=True),
+            schema='guidebook')
+
+        op.add_column(
+            table,
+            sa.Column('mtb_up_rating',
+                      sa.Enum('M1', 'M2', 'M3', 'M4', 'M5',
+                              name='mtb_up_rating', schema='guidebook'),
+                      nullable=True),
+            schema='guidebook')
+
+        op.add_column(
+            table,
+            sa.Column('mtb_down_rating',
+                      sa.Enum('V1', 'V2', 'V3', 'V4', 'V5',
+                              name='mtb_down_rating',
+                              schema='guidebook'),
+                      nullable=True),
+            schema='guidebook')
+    # update the columns useing a sql script
+    connection = op.get_bind()
+    raw_file = join(dirname(__file__), '24f8da659c78_outings.sql')
+    f = codecs.open(raw_file, encoding='utf-8')
+    content = f.read()
+    connection.execute(text(content))
```

Comment:
Do you really need to explicitly call the connection object?
What about simply
```
op.execute(text(content))
```
as in https://github.com/c2corg/v6_api/blob/master/alembic_migration/versions/8e0f02982746_update_avalanche_slopes.py#L66-L70

Reply:
With all the INSERT queries coming, would it makes sense to add BEGIN/COMMIT/ROLLBACK statements in case something goes wrong?